src/aws-cpp-sdk-core/include/aws/core/config/AWSProfileConfig.h

```diff
@@ -19,6 +20,24 @@ namespace Aws
         class Profile
         {
         public:
+            /*
+             * Data container for service endpoints.
+             */
+            class Services
+            {
+            public:
+                Services() = default;
```

Comment:
why do we need a default constructor? seems like we should only have the constructor with endpoints and name